package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "pusher",
-  "description": "Node.js client to interact with the Pusher REST API",
-  "version": "2.2.0",
+  "description": "Node.js client to interact with the Pusher Channels REST API",
+  "version": "2.2.1",
```

Comment:
I'm not sure you actually need to make a release for these changes to be picked up on https://www.npmjs.com/.

Reply:
hm. I'll try to merge the PR without changing the version and see how it goes.